content/news/playtest-20160930.md

```diff
@@ -0,0 +1,33 @@
+We are pleased to release the second playtest of this development cycle!
+
+Over 5,000 people have tried out the previous playtest, which is about twice as many as usual. Thanks to everyone one of those people for testing and contributing back! Stable releases would not be possible without you. We now count on you to check out this playtest as rigorously as the first one. With any luck, this might be the last chance to report issues before the stable release, so don't miss it!
+
+As usual we have fixed quite a few bugs, many of them very visible and annoying. These include:
+
+* Servers becoming unusable in various ways (grey ping, kicking players on joining, etc.).
+* Some hotkeys not working (closing the ingame chat window, navigation hotkeys in the TD mod).
+* Aircraft reloading behaviour still having problems.
+* Support power speech notifications missing.
+* Crashes for various reasons.
+* Issues in mission scripts.
+
+Additionally, we have tweaked the balance of the RA mod some more. Most notably, the Soviet missile submarine now has an AA capability, so be sure to give that one a good workout.
+
+<div style="text-align:center" markdown="1">
+![Mission briefing shown in the lobby chat](/images/news/20160930-ra-lobby-mission-briefing.png)
+
+Selecting a multiplayer mission will now show the mission briefing in the lobby chat.
+</div>
+
+To see the complete list of fixes and changes, have a look at the [complete changelog](https://github.com/OpenRA/OpenRA/wiki/Changelog/HASHHASHHASH).
+
+Please be aware of the following two things when taking this playtest out for a spin:
+
+1. Because we have changed which assets are copied, all players will need to reinstall (or redownload) the assets for this playtest. These will be used for the future playtests and releases, so only needs to be done once.
```

Comment:
Can you please tweak the wording here to make it clear that this is for release -> playtest, not last playtest -> this playtest?